meta-resin-common/recipes-support/resin-hostname/resin-hostname/resin-hostname

```diff
@@ -5,7 +5,8 @@ set -e
 . /usr/sbin/resin-vars
 
 if [ -z "$CONFIG_HOSTNAME" ]; then
-    CONFIG_HOSTNAME=$(jq -r '.uuid //empty' $CONFIG_PATH)
+    # take just the first 7 characters
+    CONFIG_HOSTNAME=$(jq -r '.uuid //empty' $CONFIG_PATH | sed -e 's/\(.......\).*/\1/')
```

Comment:
For a more not-so-dotted look you could try:

sed -e 's/\(.\{7\}\).*/\1/'

But I really have no preference